### ISSUE TITLE
application in musicvae 

### ISSUE THREAD

**Issue by cristianmtr** [brain-research/acai#10]
Hey

Could this be applied to MusicVAE? I'm not sure about whether this can help with interpolation of sequences of MIDI melodies. Any advice on where in the code I could look for that?

Thanks

**Comment by craffel** [comment]
Hi, potentially, the main difficulty is that MusicVAE outputs in a categorical (one-hot) space. Since ACAI operates on reconstructions and interpolants, and not raw data directly, you could probably feed the logits into the critic. MusicVAE already interpolates reasonably well though, so I'm not sure if this would confer a big advantage.
